install.sh

```diff
@@ -886,7 +886,11 @@ setup_etcd_user() {
 update_kernel_params() {
     for param in vm.panic_on_oom=0 kernel.panic=10 kernel.panic_on_oops=1 kernel.keys.root_maxbytes=25000000 vm.overcommit_memory=1; do
         ${SUDO} sysctl -w ${param}
-        echo ${param} | ${SUDO} tee -a /etc/sysctl.d/local.conf >/dev/null
+        # check if we've already written these values and only write them
+        # if we haven't previously.
+        if ! grep -Fxq ${param} /etc/sysctl.d/local.conf >/dev/null; then
```

Comment:
Could this be `grep -Fxqv` instead of `! grep -Fxq`?

Reply:
Fails due to the file not existing at the outset.